compose.js

```diff
@@ -50,10 +50,35 @@ function asyncify(f) {
   }
 }
 
+
+// This accepts an array of arrays and outputs a combination of their layers.
```

Comment:
If anyone knows the name for this algorithm I'm sure there's an idiomatic way to do it. My function works fine but I'd prefer to reuse an algorithm that's been more than an hour old. :~)